File: Python/Question Practice Python/Fibonacci.py
def Fibonacci(num):
    if num <= 1:
        return num
    else:
        return Fibonacci(num - 1) + Fibonacci(num - 2)

limit = int(input("Enter the Limit: "))
print("Fibonacci Series: ", end=" ")
for i in range(limit):
    print(Fibonacci(i), end=" ")

File: Java/Question Practice Java/Fibonacci.java
import java.util.Scanner;
public class file{
    public static int Fibonacci(int lim){
        if (lim <= 1) return lim;
        return Fibonacci(lim - 1) + Fibonacci(lim - 2);
    }
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the Limit: ");
        int limit = input.nextInt();

        System.out.print("Fibonacci Series: ");
        for (int i = 0; i < limit; i++) System.out.print(Fibonacci(i) + " ");
    }
}